src/main/docs/guide/serviceDiscovery.adoc

```diff
@@ -37,6 +37,23 @@ consul:
         defaultZone: "${CONSUL_HOST:localhost}:${CONSUL_PORT:8500}"
 ----
 
+When using Service Discovery, Micronaut will register the service in Consul using the name defined in `micronaut.application.name`.
+If the application also uses an HTTP Server (Netty, Tomcat,...), Micronaut will register the application with the same
+name and a different port in Consul. In case you want to use a different name for the GRPC service in Consul:
+
+[source,yaml]
+----
+micronaut:
+    application:
+        name: greeter # <1>
+grpc:
+    server:
+        instance-id: 'service-grpc' # <2>
+----
+<1> The HTTP port will be registered in Consul with the name `greeter`
+<2> The GRPC port will be registered in Consul with the name `hello-grpc`
```

Comment:
This documentation appears to be incorrect

Reply:
Sorry, bad copy-paste.